File: server/php/src/Application/Domain/Payments/Card.php
<?php

namespace OnlinePayments\ExampleApp\Application\Domain\Payments;

class Card
{
    public ?string $number = null;
    public ?string $holderName = null;
    public ?string $verificationCode = null;
    public ?string $expiryMonth = null;
    public ?string $expiryYear = null;

    public function __construct(
        ?string $number = null,
        ?string $holderName = null,
        ?string $verificationCode = null,
        ?string $expiryMonth = null,
        ?string $expiryYear = null
    ) {
        $this->number = $number;
        $this->holderName = $holderName;
        $this->verificationCode = $verificationCode;
        $this->expiryMonth = $expiryMonth;
        $this->expiryYear = $expiryYear;
    }
}

File: server/python/app/application/domain/payments/card.py
from typing import Optional
from pydantic import BaseModel, ConfigDict
from pydantic.alias_generators import to_camel

class Card(BaseModel):
    model_config = ConfigDict(alias_generator=to_camel, populate_by_name=True)

    number: Optional[str] = None
    holder_name: Optional[str] = None
    verification_code: Optional[str] = None
    expiry_month: Optional[str] = None
    expiry_year: Optional[str] = None